core/driver/sources/src/main/resources/reference.conf

```diff
@@ -0,0 +1,81 @@
+# Default configuration for the org.virtuslab.ideprobe.config.IdeProbeConfig
+probe {
+
+  # intellij is for org.virtuslab.ideprobe.config.IntellijConfig - default values below
+  intellij {
+    plugins = []
+    version {
+      build = "212.5080.55"
+      release = "2021.2.1"
+    }
+    # path = "" // use this config with proper path string if you want to use non-default IntellijConfig
+  }
+
+  # workspace is for Option[org.virtuslab.ideprobe.config.WorkspaceConfig] - default None - comments added as a reference
+  # workspace {
+  #   path { // for org.virtuslab.ideprobe.dependencies.Resource
+  #     uri {
+  #       str = ""
+  #     }
+  #     path {} // for java.nio.file.Path interface - choose proper implementation and add proper parameter names
+  #     cause {
+  #       s = ""
+  #     }
+  #   }
+  #   existing {} // for java.nio.file.Path interface - choose proper implementation and add proper parameter names
+  #   branch = ""
+  #   tag = ""
+  #   commit = ""
+  # }
+
+  # resolvers id for org.virtuslab.ideprobe.config.DependenciesConfig.Resolvers - default values below
+  resolvers {
+    intellij {
+      repositories = []
+    }
+    plugins { }
+    jbr {
+      repositories = []
```

Comment:
Same with JBR... IIRC `https://cache-redirector.jetbrains.com/intellij-jbr/jbr_dcevm-[major]-[platform]-x64-b[minor].tar.gz` is the the default

Reply:
it should be done with https://github.com/VirtusLab/ide-probe/issues/253

Reply:
Pls note down all specific things to be done like that, e.g. as checkboxes on PR description... see e.g. https://github.com/VirtusLab/git-machete-intellij-plugin/issues/1004 as an _inspiration_ ;)